templates/commands/render.go

```diff
@@ -116,6 +117,7 @@ func (r *Render) Flags() *cli.FlagSet {
 		Example: "path/to/spec.yaml",
 		Target:  &r.flagSpec,
 		Default: "./spec.yaml",
+		Predict: predict.Files("*"),
```

Comment:
I'm not sure about this. `--spec` is relative to the downloaded template directory, so I don't think this will autocomplete correctly. It might not be possible to do any cleverness here.

Reply:
Done